pkg/deploy/labels.go

```diff
@@ -12,10 +12,21 @@
 package deploy
 
 import (
+	"strings"
+
 	orgv1 "github.com/eclipse/che-operator/pkg/apis/org/v1"
 )
 
 func GetLabels(cr *orgv1.CheCluster, component string) (labels map[string]string) {
 	labels = map[string]string{"app": DefaultCheFlavor(cr), "component": component}
 	return labels
 }
+
+func MergeLabels(labels map[string]string, additionalLabels string) {
```

Comment:
Sorry, that's enougth to have one additionalLabels ? If it is only one than it should be called additionalLabel. If we need list, than we should use array instead of string.

Reply:
Oh, I see, comma separated/